airbyte-server/src/main/java/io/airbyte/server/ServerApp.java

```diff
@@ -210,13 +211,28 @@ public static void main(String[] args) throws Exception {
       LOGGER.info(String.format("Setting Database version to %s...", airbyteVersion));
       jobPersistence.setVersion(airbyteVersion);
     }
-    final Optional<String> airbyteDatabaseVersion = jobPersistence.getVersion();
+
+    Optional<String> airbyteDatabaseVersion = jobPersistence.getVersion();
```

Comment:
nit: put this in a separate function `runAutomaticMigration`.

would make the main function a tad easier to read.